v4/mobile/react-native/FastLink/patches/react-native+0.62.2.patch

```diff
@@ -0,0 +1,39 @@
+diff --git a/node_modules/react-native/React/CxxBridge/RCTCxxBridge.mm b/node_modules/react-native/React/CxxBridge/RCTCxxBridge.mm
+index 18b0e09..6e676f9 100644
+--- a/node_modules/react-native/React/CxxBridge/RCTCxxBridge.mm
++++ b/node_modules/react-native/React/CxxBridge/RCTCxxBridge.mm
+@@ -768,7 +768,7 @@ - (void)registerExtraLazyModules
+ #endif
+ }
+ 
+-- (NSArray<RCTModuleData *> *)_initializeModules:(NSArray<id<RCTBridgeModule>> *)modules
++- (NSArray<RCTModuleData *> *)_initializeModules:(NSArray<Class> *)modules
```

Comment:
Fixes the build error:

<img width="719" alt="Screenshot 2024-07-15 at 18 35 35" src="https://github.com/user-attachments/assets/87abb6eb-43c2-420e-92e9-095b5ad7ab7b">